custom_components/multiscrape/form.py

```diff
@@ -150,11 +154,20 @@ async def async_submit(self, main_resource):
         if self._submit_once:
             self._should_submit = False
 
+        await self._scraper.set_content(response.text)
```

Comment:
Only required in case of variables, right?

Reply:
Yes. Per other comment we are not initializing scraper if there are no variables, so checking here if scraper is initialized. It's bettercheck anyways, if we decide to use scraper in the form submit for something else other than variables.